### DISCUSSION TITLE
How to deserialize specific complex object from yaml file

### DISCUSSION THREAD

**Discussion by rdkumavat1** [aaubry/YamlDotNet#945] [General]
Can you please share link of sample code that shows how to deserialize specific complex object from yaml file

**Comment by EdwardCooke** [comment]
Is this what you are talking about?

https://github.com/aaubry/YamlDotNet/wiki/Samples.DeserializeObjectGraph

**Reply by rdkumavat1** [reply]
I don't have nested classes, having independent classes with properties and the same is added in yaml. I wan to read only specific object structure from yaml
